services/web/client/source/class/osparc/data/model/Node.js

```diff
@@ -1131,6 +1131,12 @@ qx.Class.define("osparc.data.model.Node", {
       osparc.data.Resources.fetch("studies", "getNode", params)
         .then(data => this.__onNodeState(data))
         .catch(err => {
+          if ("status" in err && err.status === 503) {
+            const interval = Math.floor(Math.random() * 5000) + 3000;
+            console.log(this.getNodeId(), "node unresponive, trying again in", interval);
+            setTimeout(() => this.__nodeState(), interval);
```

Comment:
should you not cap this retry to a max amount of times?

Reply:
idk, @sanderegg what do you think?

Reply:
Is that for the on start? or also for healtchecking?
for on start I would say a few minutes. for healthchecking, I guess this could remain.

Maybe in case you get a 401 then I would stop. But for now this will not happen.

Reply:
Retrying 5 times max